src/components/webinars/Webinar.js

```diff
@@ -15,7 +16,11 @@ export default ({ webinar }) => (
       </div>
 
       <div className={styles.body}>
-        <p>{webinar.webinarDate}</p>
+        <p>
+          <time dateTime={webinar.webinarDate}>
+            {format(new Date(webinar.webinarDate), 'EEEE, MMMM d, yyyy, h:mm a')}
```

Comment:
This format string is obviously working, but I don't really get it — `EEEE` isn't included in the `date-fns` docs and it looks like you may want an upper case `D`. 

Based on what I'm seeing on those docs, the string should be `dddd, MMMM D, YYYY, h:mm a`.

Also, I'm assuming marketing requested that you include the time, but I'm not sure it's helpful without a timezone attached. What do you think?

Reply:
Here are the date-fns docs I was referencing btw
https://date-fns.org/v1.28.5/docs/format

Reply:
You linked to v1 docs; I'm using v2.

https://date-fns.org/v2.14.0/docs/format

Reply:
`EEEE` gives us the day of the week
`d` gives us day of the month; `1, 2, 3, 4, etc`

Reply:
ah that makes sense, sorry about that

Reply:
As for the timezone, not sure I'm going to be able to reliably show the timezone considering daylight savings time. I can show the UTC offset, but I'm not sure if that's helpful or if most people understand it? I thought the fact that the time is converted based on the user's system time zone would be fine.

The other option I was considering is put a new field in Contentful for time and allow a string which we instruct to be entered in Pacific time and then we can hardcode it `July 21, 2020, 9:00 AM Pacific`. Everyone would see the time in Pacific. I wasn't sure if this was better or worse of having the time automatically change based on the user.

Reply:
date-fns has a timezone option for `format`, I'll try to add it.

Reply:
I think an offset or always setting it to Pacific would be a little more clear. To me at least, it didn't feel obvious that the time was based on my computer's time zone. If date-fns has a clean way to format the user's local time zone, that seems like the best option to me.

Reply:
Added a separate field for time and a new field for timezone in Contentful.

<img width="815" alt="image" src="https://user-images.githubusercontent.com/1479563/87474810-b274a300-c5f1-11ea-9102-69a84eb41a07.png">

Reply:
date-fns was removed; relying on Gatsby/GraphQL `formatString` method